src/Database/Api/DatabaseOptions.cs

```diff
@@ -15,12 +15,19 @@
  */
```

Comment:
Should this file be in src/Configuration/... ?

Reply:
moved